services/static-webserver/client/source/class/osparc/workbench/DiskUsageIndicator.js

```diff
@@ -0,0 +1,180 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2020 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+/**
+ * Widget that shows an image well centered and scaled.
+ * | _________________________________ |
+ * | XXXXXXXXXXXX______ X GB__________ |
+ * |___________________________________|
+ */
+qx.Class.define("osparc.workbench.DiskUsageIndicator", {
+  extend: qx.ui.core.Widget,
+
+  construct: function() {
+    this.base(arguments);
+    const lowDiskSpacePreferencesSettings = osparc.Preferences.getInstance();
+    this.__lowDiskThreshold = lowDiskSpacePreferencesSettings.getLowDiskSpaceThreshold();
+    this.__lastDiskUsage = {};
+
+    this._setLayout(new qx.ui.layout.VBox());
+
+    // Subscribe to disk space threshold - Default 5GB
+    lowDiskSpacePreferencesSettings.addListener("changeLowDiskSpaceThreshold", e => {
+      this.__lowDiskThreshold = e.getData();
+      this.__updateDiskIndicator();
+    }, this);
+  },
+
+  properties: {
```

Comment:
This guy could use at least one more property `usedSpace` (next prs)

Reply:
👍